Node/src/TeamsChatConnector.ts

```diff
@@ -182,27 +189,41 @@ export class TeamsChatConnector extends builder.ChatConnector {
     for (var event of events) {
       let invoke = <IInvokeEvent>event;
```

Comment:
Nit: 
- This should be inside the if
- if should be if (event.type === 'invoke')